web/themes/custom/pineapple/sass/layouts/nodes/_recipe.scss

```diff
@@ -99,3 +98,96 @@
     }
   }
 }
+
+.recipe--teaser {
+  display: flex;
+  flex-direction: column;
+  background-color: #f1f0f4;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+
+  .recipe-card-wrapper {
+
+    .teaser-image-container {
+      position: relative;
+
+      .highlighted-label {
+        background-color: transparentize($primary, 0.4);
+        color: $white;
+        text-transform: uppercase;
+        text-align: left;
+        margin-top: $unit * 3;
+        float: right;
+        position: absolute;
+        right: 0;
+        top: 0;
+        z-index: 1;
+      }
+
+      img {
+        width: 100%;
+        object-fit: cover;
+        height: 220px;
```

Comment:
You don't need to set this height, the image will take height once you properly enable the image style

Reply:
done